processing/src/main/java/io/druid/query/DruidMetrics.java

```diff
@@ -59,6 +61,7 @@ public static int findNumComplexAggs(List<AggregatorFactory> aggs)
   public static <T> ServiceMetricEvent.Builder makePartialQueryTimeMetric(Query<T> query)
   {
     return new ServiceMetricEvent.Builder()
+        .setDimension(VERSION, Strings.nullToEmpty(DruidMetrics.class.getPackage().getImplementationVersion()))
```

Comment:
I think it supports it, the underlying userDims in ServiceMetricEvent is just a simple Map<String, Object>